File: U3P2C#/U3P_16.cs
using System;

namespace MyApp // Note: actual namespace depends on the project name.
{
    internal class U3P_16
    {
        static void Main(string[] args)
        {
            int c = 1, horas, pago, sueldo;

            Console.WriteLine("captura las horas trabajadas");
            horas = int.Parse(Console.ReadLine());
            Console.WriteLine("captura el pago por hora");
            pago = int.Parse(Console.ReadLine());
            sueldo = horas * pago;
            Console.WriteLine("tu sueldo es: " + sueldo);
            c++;
        }
    }
}

File: U3P2J/U3P_16.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.u3p_16;
import java.util.Scanner;
/**
 *
 * @author LaboratorioLTI2
 */
public class U3P_16 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int c=1, horas, pago, sueldo;
        
        System.out.println("captura las oras trabajadas");
        horas= sc.nextInt();
        System.out.println("captura el pago x por hora");
        pago= sc.nextInt();
        sueldo=horas*pago;
        System.out.println("tu sueldo es:"+sueldo);
        c++;
    }
}
